staff/acct/note

```diff
@@ -8,12 +8,16 @@ import shutil
 import sys
 from datetime import date
 
+from ocflib.misc.shell import yellow
+
 USER_INFO = '/home/s/st/staff/User_Info'
 USER_INFO_BAK = '/tmp/User_Info.bak.' + getpass.getuser()
 
 def main():
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument('-u', '--user', required=True, help='The username')
+    parser.add_argument('-t', '--topic',
```

Comment:
Perhaps add `choices=['quota', 'sorry', 'password', 'mail', 'sharing', 'website', 'h0zer', 'info']` or something like that, make the choices explicit?

https://docs.python.org/3/library/argparse.html#choices